meta-lmp-base/recipes-support/compose-apps-early-start/compose-apps-early-start.bb

```diff
@@ -25,3 +25,4 @@ do_install() {
 }
 
 FILES:${PN} += "${systemd_system_unitdir}/*.service"
+RDEPENDS:${PN} = "aktualizr-lite-apps docker-compose"
```

Comment:
`docker-compose` is already a runtime dependency of  `aktualizr-lite-apps` so is not need here again.

Reply:
Thank you, fixed .